k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -476,20 +472,20 @@ printf "\n"
 
 namespace=$(parseSolution 'solution.namespace')
 namespace=$(echo "${namespace}" | cut -f2 -d'>')
-parsed_node_output=$(parseSolution 'solution.nodes.node*.name')
+### TODO CORTX-29861 Revisit for best way to parse this YAML section with new schema references
+parsed_node_output=$(yq e '.solution.storage_sets[0].nodes' -o=c "${solution_yaml}")
 
 # Split parsed output into an array of vars and vals
-IFS=';' read -r -a parsed_var_val_array <<< "${parsed_node_output}"
+IFS=',' read -r -a parsed_node_array <<< "${parsed_node_output}"
```

Comment:
```suggestion
IFS=',' read -r -a parsed_node_array < <(yq '.solution.storage_sets[0].nodes' --output-format=csv "${solution_yaml}")
```
Can also remove the need for the temporary string.